docs/auto-tag-products-with-incoming-inventory/README.md

```diff
@@ -0,0 +1,55 @@
+# Auto-tag products with incoming inventory
+
+Tags: Auto-Tag, Bulk, Inventory, Products
```

Comment:
add Transfer tag, you'll need to add here as well: https://github.com/lightward/mechanic-tasks/blob/main/lib/task_schema.json